pkgs/development/python-modules/click-shell/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, click
+, pytestCheckHook
+, pytest-click
+}:
+
+buildPythonPackage rec {
+  pname = "click-shell";
+  version = "2.1";
+  format = "setuptools";
+
+  # PyPi release is missing tests
+  src = fetchFromGitHub {
+    owner = "clarkperkins";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    sha256 = "0lla3dvaw7mxm96352gwgv4qp3pjh27lr8k1ra2y65ij1p7502p1";
```

```suggestion
    hash = "";
```

Use hash and provide an SRI-Hash instead.